app/assets/stylesheets/post.css

```diff
@@ -21,11 +21,19 @@
   flex-direction: column;
```

Comment:
- [ ] It would be better to ensure that our PR looks professional. We can format it nicely. Kindly checkout this [link](https://www.markdownguide.org/), it will help us make our repository professional

Reply:
Thank you for the feedback, issue has been fixed.